extensions/SiteMapIndex/Extension.pm

```diff
@@ -90,9 +90,6 @@ sub install_before_final_checks {
     }
 
     return if (correct_urlbase() ne 'https://bugzilla.mozilla.org/');
-
-
-    $self->_fix_robots_txt();
```

Comment:
1) You need to remove the _fix_robots_txt() function as well.
2) You need to create an extensions/SiteMapIndex/template/en/default/hook/global/robots-end.txt.tmpl file that adds in the stuff that the _fix_robots_txt() function would normally do.